packages/replay-next/components/sources/SourceSearch.tsx

```diff
@@ -142,7 +142,14 @@ export default function SourceSearch({
           <button
             className={styles.ResultsIconButton}
             data-test-id="SourceSearchClearButton"
-            onClick={searchActions.disable}
+            onClick={() => {
+              searchActions.disable();
+
+              const container = containerRef.current;
+              if (container) {
+                container.focus();
+              }
```

Comment:
This change restores focus on the Source (so if you type CMD+F again, the source search will re-open). It's just a little tweak I noticed while poking around.